assets/jsons/translations/en.json

```diff
@@ -455,7 +456,8 @@
                     "EXIT": "BeatSaber stopped abruptly, tries to check the files.",
                     "OCULUS_LIB_NOT_FOUND": "Check that the Oculus application is properly installed, and that the libraries are correctly defined in Oculus.",
                     "PROTON_NOT_SET": "Set the Proton path in settings.",
-                    "PROTON_NOT_FOUND": "Set the Proton path in settings."
+                    "PROTON_NOT_FOUND": "Set the Proton path in settings.",
+                    "ORIGINAL_OCULUS_NOT_INSTALLED": "Download the original Beat Saber from Oculus before opening."
```

Comment:
I wonder if `Beat Saber must be installed from the Oculus app before opening` would be better 🤔

Reply:
Yep, agreed